src/scss/generic/_typography.scss

```diff
@@ -18,14 +18,10 @@ body {
   }
 }
 
-[hidden] {
-  display: none;
-}
-
 [data-hidden] {
-  position: absolute;
-  top: -999em;
-  opacity: 0;
+  height: 3em;
+  position: relative;
+  z-index: -999;
```

Comment:
Would this also work with `z-index: -1`? I'd prefer that so that it is less magic.

And maybe if we just use this to hide the LP-specific password field, we could make it very specific to that? Perhaps by using `#off-screen-field-password` or `[id="off-screen-field-password"]` (if we would want less specificity). I think that will also make it easier to remove when we can.

Reply:
It would, but I usually use -999 to indicate that it should always be below everything. If you'd prefer -1, I can change it to that.

Reply:
I was thinking what if something else would need to go below that… but nah, not likely at all for negative values, who stacks invisible things? Let’s keep it `-999`